packages/contracts-bedrock/scripts/deploy/ChainAssertions.sol

```diff
@@ -433,8 +434,13 @@ library ChainAssertions {
         // Check that the contract is initialized
         assertSlotValueIsOne({ _contractAddress: address(superchainConfig), _slot: 0, _offset: 0 });
 
-        require(superchainConfig.guardian() == _cfg.superchainConfigGuardian());
-        require(superchainConfig.paused() == _isPaused);
+        if (_isProxy) {
+            require(superchainConfig.guardian() == _cfg.superchainConfigGuardian());
+            require(superchainConfig.paused() == _isPaused);
+        } else {
+            require(superchainConfig.guardian() == address(0));
+            require(superchainConfig.paused() == false);
```

Comment:
I'd prefer to stay consistent with the rest of the `ChainAssertion` which have no reason string, but defer to reviewers.

Reply:
The lack of reason strings in ChainAssertions does bug me, because when one fails it's harder to track it down. Having a reason string makes it unambiguous, so I always prefer reason strings, but not a blocker

Reply:
Let's skip adding reason strings, going to go on a crusade to fix this later...